test_data/form_response0.yaml

```diff
@@ -0,0 +1,52 @@
+title: UK Air Quality Reanalysis
+description: 'This reanalysis has been produced  by the Atmospheric Dispersion and
+  Air Quality Group at the Met Office using Air Quality in the Unified Model (AQUM).
+  The reanalysis begins in January 2003 and will be maintained as a rolling 20 year
+  archive. The data covers the UK at a horizontal resolution of 0.11 degree (~12 km),
+  a vertical resolution of 63 model levels (model top ~40 km) and an hourly temporal
+  resolution. The reanalysis dataset also contains bias corrected surface concentrations
+  of some pollutants. Observations from the Automatic Urban and Rural Network (AURN),
+  London Air Quality Network (LAQN) and additional UK local authority monitoring stations
+  are used to produce the bias corrected surface concentrations.
+
+
+  Further information can be found at https://www.ukcleanair.org/storage/met-office-research/'
+authors:
+- firstname{i}: Eleanor
+  surname{i}: Smith
+- firstname: .nan
+  surname: .nan
```

Comment:
Right, I see.  Where do I find $TMPDIR?

Reply:
`$TMPDIR` is just an environment variable that should always be available because it's set by some system config. Can `cd $TMPDIR` without worrying about where it'll take you exactly (though on our setup it should be `/var/tmp`)

Reply:
wow, that was a labyrinth in itself.  i should have updated the yaml file though and have now pushed it up (on the basis that we want to keep the commit history simple despite my branch error).